src/gm.erl

```diff
@@ -51,10 +60,25 @@ mogrify(File, Options) ->
   TemplateOpts = [{file, File}],
   exec_cmd(Template, TemplateOpts, Options).
 
+%% Montage
+montage(Files, Converted, Options) ->
+  Template = "montage {{options}} :input_file :output_file",
+  TemplateOpts = [{input_file, string:join(Files, "' '")}, {output_file, Converted}],
+  exec_cmd(Template, TemplateOpts, Options).
+
+%% Version
+version() ->
+  Template = "version",
+  exec_cmd(Template).
+
 %% =====================================================
 %% INTERNAL FUNCTIONS
 %% =====================================================
 
+%% Run an os:cmd based on a template without options
+exec_cmd(Template) ->
+  os:cmd(lists:concat(["gm ",Template])).
```

Comment:
In my opinion, this you said improves readability. In fact, I didn't add commas to maintain consistency with other functions, for example on line 87 of the same file. Could we add spaces in all similar cases?